x-pack/plugin/sql/qa/security/src/test/java/org/elasticsearch/xpack/sql/qa/security/JdbcSecurityIT.java

```diff
@@ -230,16 +230,8 @@ public void expectUnknownColumn(String user, String sql, String column) throws E
 
         @Override
         public void checkNoMonitorMain(String user) throws Exception {
-            // Most SQL actually works fine without monitor/main
-            expectMatchesAdmin("SELECT * FROM test", user, "SELECT * FROM test");
-            expectMatchesAdmin("SHOW TABLES LIKE 'test'", user, "SHOW TABLES LIKE 'test'");
-            expectMatchesAdmin("DESCRIBE test", user, "DESCRIBE test");
-
-            // But there are a few things that don't work
-            try (Connection es = es(userProperties(user))) {
-                expectUnauthorized("cluster:monitor/main", user, () -> es.getMetaData().getDatabaseMajorVersion());
-                expectUnauthorized("cluster:monitor/main", user, () -> es.getMetaData().getDatabaseMinorVersion());
-            }
+            // Without monitor/main the JDBC driver - ES server version comparison doesn't take place, which fails everything else
```

Comment:
Why remove the test ? it's fairly important in checking security credentials...

Reply:
Every call to any such command will fail with unauthorized exception and at its base will be the connection creation failure, thus the replacement with the connection creation. I will re-add them back.